client/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "client-3rd-party",
+  "version": "0.0.1",
+  "description": "Used to manage 3rd party libraries for the  client side.",
+  "license": "GPL-3.0-or-later",
+  "scripts": {
+    "build": "webpack"
+  },
+  "dependencies": {
+    "@sentry/browser": "^8.24.0"
```

I think we should pin exact version here:

```suggestion
    "@sentry/browser": "8.24.0"
```

This way the webpack build will be reproducible. The update pull requests will be automatically done by renovate.